DPGAnalysis/HcalTools/plugins/CMTRawAnalyzer.cc

```diff
@@ -1940,14 +2421,15 @@ void CMTRawAnalyzer::endJob() {
   h_nls_per_run->Fill(float(lscounterrun));
   h_nls_per_run10->Fill(float(lscounterrun10));
 
+  /*
```

Comment:
Why these chunks of code have been commented out?
If not needed, can they be removed?

Reply:
If you want I can remove them. They are legacy code